### DISCUSSION TITLE
Gibberish characters on some users' machines

### DISCUSSION THREAD

**Discussion by EricFleet** [stephanrauh/ngx-extended-pdf-viewer#2299] [Q&A]
<img width="677" alt="Gibberish" src="https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/22297741/9c9077c7-b24c-422c-8e7f-d6234b455d2c">

I am including a screenshot of a sample file. Unfortunately, I cannot include the entire file. This happens on some user's machine, but not all. The top left is an address, which displays fine on my machine but no on this user's machine.

We recently upgraded to version 19 along with an Angular 17 upgrade and started getting reports on this. I am assuming I missed something in the release notes, but after a review I still don't see anything. It is challenging as I cannot reproduce it on my machine or colleague's machines, but have confirmed the issue on one user's machine. Interestingly, if I save the PDF on their machine and have them open it in Acrobat, it renders fine.

Any help is appreciated.

**Comment by stephanrauh** [comment]
That's weird. It's especially weird if it happens only on some machines. I'm not aware of any changes of the rendering engine in version 19.

Please check if there are any error messages or warnings in the browser console. The bug looks like there's a problem with loading the fonts. 

Maybe - just maybe - the problem is gone after deactivating the cache. The simplest way to do it is to open the network tab and the check the "disable cache" checkbox. At least, it's easy for developers - most users probably need your help with it.

If the error still doesn't go away, try to open the document

- in the pdf.js viewer showcase: https://mozilla.github.io/pdf.js/web/viewer.html
- in Firefox (because it's using the same PDF rendering engine).
If the error shows in either of them, I'm lucky, because then it's a problem of the root engine. In that case, please open a bug ticket here: https://github.com/mozilla/pdf.js/issues

**Comment by EricFleet** [comment]
Thank you. I am going to see if we can get time with one of the people affected to try this out.

**Reply by MarianneAK** [reply]
Hi Eric, were you able to solve this issue ? I have your exact same problem 😞

**Comment by AnejVovcak** [comment]
I have started a [new](https://github.com/stephanrauh/ngx-extended-pdf-viewer/discussions/2854) disccusion since I am not sure if I get the same issue exactly.
